### PR TITLE
Post UI Refresh: Mobile fixes + minor SDK updates

### PR DESCRIPTION
**Pull request by koredefashokun**
## Description
This PR fixes a number of mobile issues that emerged as a result of changes included in the UI Refresh. It also includes newer design updates to the market and leaderboard pages.

It also contains a number of file cleanups, and moves some functions to the SDK. However, none of these functions are linked to the frontend.

## Related issue
#2342 

## Motivation and Context
This PR is part of a number of Post UI Refresh changes.

## How Has This Been Tested?
N/A

### CODE REVIEW

**Review by avclarke — COMMENTED** [review]
Looks good, just a couple of minor suggestions.

File: sdk/services/staking.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import {
+	AGGREGATE_ASSET_KEY,
+	FUTURES_ENDPOINT_OP_MAINNET,
+	KWENTA_TRACKING_CODE,
+} from 'sdk/constants/futures';
+import { SECONDS_PER_DAY } from 'sdk/constants/period';
+import { DEFAULT_NUMBER_OF_FUTURES_FEE } from 'sdk/constants/staking';
+import { getFuturesAggregateStats, getFuturesTrades } from 'sdk/utils/subgraph';
+
+export default class StakingService {
```
[the remark this answers is not in the document]
Comment (koredefashokun) [reply sdk/services/staking.ts]: Merged this with the token service in the latest commit.

File: sdk/services/stats.ts [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
 
+import { REQUIRES_L2 } from 'sdk/common/errors';
+import { FUTURES_ENDPOINT_OP_MAINNET } from 'sdk/constants/futures';
+import { ETH_UNIT } from 'sdk/constants/transactions';
+import { AccountStat, FuturesStat } from 'sdk/types/stats';
+import { weiFromWei } from 'sdk/utils/number';
+import { truncateAddress } from 'sdk/utils/string';
+import { getFuturesStats } from 'sdk/utils/subgraph';
+
+const mapStat = (stat: FuturesStat, i: number) => ({
```
[the remark this answers is not in the document]
Comment (koredefashokun) [reply sdk/services/stats.ts]: Moved in the latest commit.

File: sdk/services/stats.ts [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+	rankText: (i + 1).toString(),
+});
+
+type LeaderboardPart = 'top' | 'bottom' | 'wallet' | 'search' | 'all';
+
+type LeaderboardResult = {
+	[part in LeaderboardPart]: AccountStat[];
+};
+
+export const DEFAULT_LEADERBOARD_DATA = {
```
[the remark this answers is not in the document]
Comment (koredefashokun) [reply sdk/services/stats.ts]: Moved.

File: sections/futures/Trade/MarketsDropdown.tsx [hunk 4]
```diff
@@ -370,7 +370,7 @@ const SearchBarContainer = styled.div`
 `;
 
 const SelectContainer = styled.div<{ mobile?: boolean; accountType?: string }>`
-	z-index: 100;
+	z-index: 40;
```
[the remark this answers is not in the document]
Comment (koredefashokun) [reply sections/futures/Trade/MarketsDropdown.tsx:373]: Done.

**Review by avclarke — APPROVED** [review]

**Review by platschi — APPROVED** [review]

**Pull request merged**